src/main/java/com/kodilla/ecommercee/UserController.java

```diff
@@ -1,4 +1,35 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.domain.UserDto;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/v1/user")
 public class UserController {
+    @RequestMapping(method = RequestMethod.GET, value = "getUsers")
+    public List<UserDto> getUsers() {
+        return new ArrayList<>();
+    }
+
+    @RequestMapping(method = RequestMethod.GET, value = "getUser")
```

Comment:
Wcześniej używaliśmy już w projekcie skróconej wersji adnotacji, dla tego przypadku GetMapping. Chyba warto to ujednolicić.